modules/ztp-lvms-installing-lvms-cli.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-lvms-installing-using-cli_{context}"]
+= Installing {lvms} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install {lvms}.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of the {rh-rhacm} Operator.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `openshift-storage` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create ns openshift-storage
+----
+
+. Create an `OperatorGroup` CR.
+
+.. Define the `OperatorGroup` CR and save the YAML file, for example, `lmvs-operatorgroup.yaml`:
++
+.Example OperatorGroup CR
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: lvms-operator-operatorgroup
+  namespace: openshift-storage
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "2"
+spec:
+  targetNamespaces:
+  - openshift-storage
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f lmvs-operatorgroup.yaml
```

Comment:
If the file name can vary, this would be a place to use a placeholder and a definition list.